ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-master.yaml

```diff
@@ -34,6 +34,13 @@ images:
 - dockerfile_path: deploy/Dockerfile.registry.upgrade-prev
   from: operator-registry-base
   to: hco-registry-upgrade-prev
+operator:
+  bundles:
+  - context_dir: deploy/olm-catalog
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/kubevirt/hyperconverged-cluster-operator:1.2.0
```

Comment:
In the meantime, we had move to image digests. Dose `pullspec` support regex or something similar?

Reply:
Unfortunately not, definitely not at the moment. For now we'll need to live with that.

Reply:
Since the substitution here *creates* digests in your output bundle there's no real reason to use digests in your CSV on disk in git, no?

Reply:
I guess having digests allows to build a bundle with just a `Dockerfile.bundle` and have predictable, deterministic deps, not depending on any build system to sub them for you? Not sure.

Reply:
> Since the substitution here creates digests in your output bundle there's no real reason to use digests in your CSV on disk in git, no?
We're not releasing from CI - just using it for test. So no.

Reply:
I guess having digests allows to build a bundle with just a Dockerfile.bundle and have predictable, deterministic deps, not depending on any build system to sub them for you? Not sure.
> The reason to use digest is to support deploy in disconnected environment. Using digests is a requirement for that from OLM.